docs/developers/log-events/esdt-events.mdx

```diff
@@ -0,0 +1,231 @@
+        {label: 'Example', value: 'Example'},
+    ]}
+>
+<TabItem value="Structure">
+
+| Field      | Value                                                                                                                        |
+|------------|------------------------------------------------------------------------------------------------------------------------------|
+| identifier | ESDTNFTTransfer                                                                                                              |
+| address    | the sender  of the token                                                                                                     |
+| topics     | `topics[0]` - token identifier base64 encoded <br/>`topics[1]` - token nonce bytes base64 encoded <br/> `topics[2]` - value bytes base64 encoded <br/> `topics[3]` - receiver address bytes base64 encoded |
+| data       | empty                                                                                                                     |
+
+</TabItem>
+<TabItem  value="Example">
+
+```json
+{
+    "identifier": "ESDTNFTTransfer",
+    "address": "erd1qqqqqqqqqqqqqpgq0tajepcazernwt74820t8ef7t28vjfgukp2sw239f3",
+    "data": null,
+    "topics": [
+        "WE1FWC1mZGEzNTU=",
+        "Aw==",
+        "LUrGxb4/2VjTAA==",
+        "0+diI4/UjUcgbmchGtML1QwheMY68dnsww9vhRPJUWg="
+    ]
+}
+```
+
+</TabItem>
+</Tabs>
+
+
+### Multi token transfer
+
+The `MultiESDTNFTTransfer` event is generated when one or multiple tokens are transferred using the built-in
+function `MultiESDTNFTTransfer`.
+
+
+<Tabs
+    defaultValue="Structure"
+    values={[
+        {label: 'Structure', value: 'Structure'},
+        {label: 'Example', value: 'Example'},
+    ]}
+>
+<TabItem value="Structure">
+
+| Field      | Value                                                                                                                           | 
+|------------|---------------------------------------------------------------------------------------------------------------------------------|
+| identifier | MultiESDTNFTTransfer                                                                                                            |
+| address    | the sender  of the token                                                                                                        |
+| topics     | `topics[0]` - token identifier <br/>`topics[1]` - token nonce base64 encoded (can be empty in case of fungible token) <br/> `topics[2]` - value bytes base64 encoded <br/> `topics[3]` - receiver address bytes base64 encoded    |
+| data       | empty                                                                                                                           |
+
+</TabItem>
+<TabItem  value="Example">
+
+```json
+{
+    "identifier": "MultiESDTNFTTransfer",
+    "address": "erd1qqqqqqqqqqqqqpgq0tajepcazernwt74820t8ef7t28vjfgukp2sw239f3",
+    "data": null,
+    "topics": [
```

Comment:
in all the examples, the `topics` field if after the `data` field, but in the Structure tab the order is reversed. Please update